Competition2017/src/competition/subsystems/feeder/commands/StopFeederCommand.java

```diff
@@ -0,0 +1,27 @@
+package competition.subsystems.feeder.commands;
+
+import competition.subsystems.feeder.SideFeederSubsystem;
+
+import xbot.common.command.BaseCommand;
+
+public class StopFeederCommand extends BaseCommand {
+
+    final SideFeederSubsystem sideFeeder;
+
+    public StopFeederCommand(SideFeederSubsystem sideFeeder) {
+
+        this.sideFeeder = sideFeeder;
+        this.requires(this.sideFeeder);
+    }
+    
+    @Override
+    public void initialize() {
+        
+    }
+
+    @Override
+    public void execute() {
+
+        sideFeeder.setFeederPower(0);
```

Comment:
I think you should leave this in execute. That way, if some nefarious command comes along and changes the power, you'll set it back.